src/osgPlugins/txp/trpage_io.h

```diff
@@ -299,7 +299,7 @@ TX_EXDECL class TX_CLDECL trpg3dPoint {
 TX_EXDECL class TX_CLDECL trpgColor {
 public:
     trpgColor(float64 r,float64 g,float64 b) {red = r; green = g; blue = b;}
-    trpgColor(void) : red(1.0), green(1.0), blue(1.0) { };
```

Comment:
Let's use the 3.6 default for this instead of the vrvantage specified default.

Reply:
Resolved.